### PR TITLE
Fix NullPointerException logging the NodeId

### PR DESCRIPTION
**Pull request by seltihood**
Hi, 
We were having some exceptions because route was null (Line 77: NodeID nodeID = route.getNodeID();) 
Could you please merge this fix?
Thanks!

### CODE REVIEW

**Comment by seltihood** [comment]
Hi, I put both commits in different pull requests in case you were not interested on the other one :)

**Comment by seltihood** [comment]
@GregDThomas Could you merge this one too? We need this because we are having NullPointers exceptions due to the route being null  when getting the variable to log it in line 77 (https://github.com/igniterealtime/openfire-hazelcast-plugin/pull/52/files#diff-0ddf0d547dd3a57d1b7933fda1715a96L77)

**Review by GregDThomas — COMMENTED** [review]
Could you address my concern with the PR as it stands - i.e. I'd would sitll like to see a warning logged if the route is null.

File: src/java/org/jivesoftware/openfire/plugin/session/ClientSessionTask.java [hunk 1]
```diff
@@ -74,10 +74,11 @@ public void run() {
             // The session is being hosted by other cluster node so log this unexpected case
             Cache<String, ClientRoute> usersCache = CacheFactory.createCache(RoutingTableImpl.C2S_CACHE_NAME);
             ClientRoute route = usersCache.get(address.toString());
-            NodeID nodeID = route.getNodeID();
-
-            logger.warn("Found remote session instead of local session. JID: " + address + " found in Node: " +
-                    nodeID.toByteArray() + " and local node is: " + XMPPServer.getInstance().getNodeID().toByteArray());
+            if (route != null && route.getNodeID() != null && XMPPServer.getInstance() != null)  {
```
[the remark this answers is not in the document]
Comment (seltihood) [reply src/java/org/jivesoftware/openfire/plugin/session/ClientSessionTask.java:77]: I think that getNodeID from route can be null. It is the getNodeID coming from  XMPPServer.getInstance() that gets a random Id. 
I will tweak the logs so if route is null we still get a log :)

**Pull request merged**